### ISSUE TITLE
FirebaseAnalytics not working 

### ISSUE THREAD

**Issue by PratikPagada** [chemerisuk/cordova-plugin-firebase-analytics#49]
Native: tried calling FirebaseAnalytics.setCurrentScreen, but the FirebaseAnalytics plugin is not installed.

vendor.js:54174 Install the FirebaseAnalytics plugin: 'ionic cordova plugin add cordova-plugin-firebase-analytics'

**Comment by yunderboy** [comment]
@chemerisuk great job on the project man!
But why was this issue closed? :)

**Comment by vromaniv** [comment]
Why is this closed? I have the same issue.

**Comment by julienkermarec** [comment]
Same issue

**Comment by BinaryBlock** [comment]
@chemerisuk , I would like to humbly agree with @yunderboy @vromaniv and @julienkermarec ... why was this issue closed? I am facing the exact same problem. I can see there are similar issues that have been closed as well, without reason (or just suggesting to update to latest version). I'm not sure what I'm missing here. 

**Any assistance you can provide would be GREATLY appreciated.** 

And as @yunderboy stated, great work on this open source project! You are doing fantastic and extremely helpful work. 👍

**Comment by julienkermarec** [comment]
Hello huys !

Finally i use the global cordova plugin : cordova-plugin-firebase (https://github.com/arnesson/cordova-plugin-firebase)

**Import :** 
import { Firebase } from '@ionic-native/firebase';

**Constructor :** 
private firebase: Firebase

**With the function :** 
this.firebase.setScreenName(page);

**Comment by chemerisuk** [comment]
> But why was this issue closed? :)
I don't use ionic and have no idea how their CLI is implemented. I remember In the past the CLI had bugs with variable argument support, people complained but a ticket here is not the right place. Now with Cocopods, well... Try to add the plugin via cordova CLI first, then if iconic CLI fails - open a ticket in their repository.

**Comment by BinaryBlock** [comment]
> a ticket here is not the right place. Now with Cocopods, well... Try to add the plugin via cordova CLI first, then if iconic CLI fails - open a ticket in their repository.

This is helpful and positive information. I see this as a valid reason for closing the ticket. At the point my/our comments were made, we had no visibility into the "why" the ticket was closed. I did not see any comments or replies. I continue to learn about [Cordova plugin development](https://cordova.apache.org/docs/en/latest/guide/hybrid/plugins/index.html) and how the Ionic CLI works in conjunction with this extremely useful package.

Thanks again for the reply and additional info. I will try to followup here if I find a viable solution. 👍
